make-it/EmMakefile

```diff
@@ -93,11 +93,10 @@ ifeq ($(TARGET), wasm32-unknown-emscripten)
 endif
 
 # NO_DYNAMIC_EXECUTION=1 Allows Vireo to be used in CSP environments where eval() is forbidden: https://github.com/kripken/emscripten/blob/master/src/settings.js#L563
-EM_OPT = $(EM_OPTFLAGS) $(TARGET_FLAG) -s NO_EXIT_RUNTIME=1 -std=c++14 -fno-exceptions --memory-init-file 0 -s NO_DYNAMIC_EXECUTION=1 -s MODULARIZE=1 -s EXPORT_NAME=VireoCreateCoreModule -s EXPORT_ES6=1 --minify 0
```

Comment:
With "ABORTING_MALLOC=0", would we get a different exception later? Why do we do this?

Reply:
Potentially gives Vireo a chance to clean-up since it has those code paths. I think it would be difficult to test out of memory situations that will always successfully clean-up beyond the path we already tested.